modules/experimental-layers/src/mesh-layer/mesh-layer.js

```diff
@@ -112,7 +115,9 @@ const defaultProps = {
   // https://en.wikipedia.org/wiki/Euler_angles
   getYaw: {type: 'accessor', value: x => x.yaw || x.angle || 0},
   getPitch: {type: 'accessor', value: x => x.pitch || 0},
-  getRoll: {type: 'accessor', value: x => x.roll || 0}
+  getRoll: {type: 'accessor', value: x => x.roll || 0},
+  getScale: {type: 'accessor', value: x => x.scale || [1, 1, 1]},
+  getTranslation: {type: 'accessor', value: x => x.translate || [0, 0, 0]}
```

Comment:
Would it be easy to add a `getMatrix` accessor that returns `null` and if it returns a value, use that instead of calculating using the other props.